jobs/JEVS_CAM_STATS

```diff
@@ -101,32 +101,23 @@ export MET_PLUS_CONF="${PARMevs}/metplus_config/${STEP}/${COMPONENT}/${VERIF_CAS
 ####################################
 # Define COMIN/COMOUT variables
 ####################################
-export COMIN=${COMIN:-$(compath.py $NET/$evs_ver)}
-export COMINhiresw=${COMINhiresw:-$(compath.py ${envir}/com/hiresw/${hiresw_ver})}
-export COMINhref=${COMINhref:-$(compath.py  href/$href_ver)}
-export COMINhrrr=${COMINhrrr:-$(compath.py ${envir}/com/hrrr/${hrrr_ver})}
-export COMINnam=${COMINnam:-$(compath.py ${envir}/com/nam/${nam_ver})}
-export COMINobsproc=${COMINobsproc:-$(compath.py ${envir}/com/obsproc/${obsproc_ver})}
-export COMINmping=${COMINmping:-$(compath.py -o $NET/$evs_ver/prep/$COMPONENT)}
-export COMINprepbufr=${COMINprepbufr:-$(compath.py  obsproc/${obsproc_ver})}
-export COMINsnow=$DCOMROOT
-export COMINspcotlk=${COMINspcotlk:-$(compath.py -o $NET/$evs_ver/prep/$COMPONENT)}
-
-if [ $VERIF_CASE = severe ] ; then
-   export COMINfcst=${COMINfcst:-$(compath.py -o $NET/$evs_ver/prep/$COMPONENT)}
-   export COMINspclsr=${COMINspclsr:-$(compath.py -o $NET/$evs_ver/prep/$COMPONENT)}
-fi
-
-
-if [ $MODELNAME = href ] && [ $VERIF_CASE = precip ] ; then
-   export COMINccpa=${COMINccpa:-$(compath.py  ccpa/${ccpa_ver})}
-   export COMINmrms=$DCOMROOT/ldmdata/obs/upperair/mrms/alaska/MultiSensorQPE
-else
-   export COMINccpa=${COMINccpa:-$(compath.py -o $NET/$evs_ver/prep/$COMPONENT)}
-   export COMINmrms=${COMINmrms:-$(compath.py -o $NET/$evs_ver/prep/$COMPONENT)}
-fi
-
-export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver}/$STEP/$COMPONENT)}
+export COMIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}
+export COMINhiresw=${COMINhiresw:-$(compath.py $envir/com/hiresw/$hiresw_ver)}
+export COMINhref=${COMINhref:-$(compath.py $envir/com/href/$href_ver)}
+export COMINhrrr=${COMINhrrr:-$(compath.py $envir/com/hrrr/$hrrr_ver)}
+export COMINnam=${COMINnam:-$(compath.py $envir/com/nam/$nam_ver)}
+export COMINobsproc=${COMINobsproc:-$(compath.py $envir/com/obsproc/$obsproc_ver)}
+export COMINccpa=${COMINccpa:-$(compath.py $envir/com/ccpa/$ccpa_ver)}
+export DCOMINmrms=${DCOMINmrms:-$DCOMROOT/ldmdata/obs/upperair/mrms/alaska/MultiSensorQPE}
+export DCOMINsnow=${DCOMINsnow:-$DCOMROOT}
+export EVSINccpa=${EVSINccpa:-$COMIN/prep/$COMPONENT}
+export EVSINmrms=${EVSINmrms:-$COMIN/prep/$COMPONENT}
+export EVSINmping=${EVSINmping:-$COMIN/prep/$COMPONENT}
+export EVSINspcotlk=${EVSINspcotlk:-$COMIN/prep/$COMPONENT}
+export EVSINfcst=${EVSINfcst:-$COMIN/prep/$COMPONENT}
+export EVSINspclsr=${EVSINspclsr:-$COMIN/prep/$COMPONENT}
+
+export COMOUT=${COMOUT:-$(compath.py -o $NET/$evs_ver/$STEP/$COMPONENT)}
 
```

Comment:
@MarcelCaron-NOAA , was the thinking here to just defined all of the COMINs, DCOMINs, and EVSINs regardless of MODELNAME or VERIF_CASE?

Reply:
Yes: after making all of the needed changes to EVSIN* and DCOMIN*, I found that the conditional expressions were unnecessary.  For example, COMINmrms became DCOMINmrms for href/precip and EVSINmrms for everything else after the change; after the change, no two jobs share the same *IN name with different values.  

But now some of these *IN variables are defined and not used (e.g., "EVSINccpa" is not used by href/precip).  If that's not desirable, I can reinsert the conditional blocks.

Reply:
OK, cool. This is fine. I don't see a problem with it. Thanks!